### PR TITLE
generate-dockerfile.py: updated to use builds from Koji

### PR DESCRIPTION
**Pull request by matusmarhefka**
* openscap, scap-security-guide and openscap-daemon can now be
  installed also by specifying build ID from Koji

### CODE REVIEW

**Comment by mpreisler** [comment]
@openscap-ci add to whitelist

File: generate-dockerfile.py [hunk 1]
```diff
@@ -71,13 +71,22 @@
...
     parser.add_argument("--base", type=str, default="fedora", help="Base image name")
-    parser.add_argument("--openscap-from-git", action="store_true", default=False,
-                        help="Use OpenSCAP from upstream instead of package")
-    parser.add_argument("--ssg-from-git", action="store_true", default=False,
-                        help="Use SCAP Security Guide from upstream instead of package")
-    parser.add_argument("--daemon-from-local", action="store_true", default=False,
-                        help="Use OpenSCAP Daemon from local working tree instead of package")
+    openscap_group.add_argument("--openscap-from-git", action="store_true",
+        default=False, help="Use OpenSCAP from upstream instead of package")
+    openscap_group.add_argument("--openscap-from-koji", type=str, default="None",
```
Comment (mpreisler) [inline generate-dockerfile.py]: why "None" and not None? default=None is already the default

**Comment by jan-cerny** [comment]
Maybe we can also check if the base image is Fedora. It doesn't make sense to install packages from Koji on other operating systems.

**Comment by mpreisler** [comment]
I also wish this was done in multiple commits. The package manager cache clean is a completely separate change.

**Comment by matusmarhefka** [comment]
Commit 3fce706 should reflect all the provided feedback. Than you.

**Comment by jan-cerny** [comment]
Looks OK. I've tested this and it generates every Dockerfile as expected. Thank you!

**Pull request merged**